tests/amd_migraphx_test/mnist/migraphx_node_test.cpp

```diff
@@ -0,0 +1,159 @@
+#include "vx_amd_migraphx.h"
+#include <cstring>
+#include <random>
+#include <fstream>
+#include <algorithm>
+#define MAX_STRING_LENGTH 100
+
+using namespace std;
+
+#define ERROR_CHECK_STATUS(status) { \
+    vx_status status_ = (status); \
+    if (status_ != VX_SUCCESS) { \
+        printf("ERROR: failed with status = (%d) at " __FILE__ "#%d\n", status_, __LINE__); \
+        exit(1); \
+    } \
+}
+
+#define ERROR_CHECK_OBJECT(obj) { \
+    vx_status status_ = vxGetStatus((vx_reference)(obj)); \
+    if(status_ != VX_SUCCESS) { \
+        printf("ERROR: failed with status = (%d) at " __FILE__ "#%d\n", status_, __LINE__); \
+        exit(1); \
+    } \
+}
+
+static void VX_CALLBACK log_callback(vx_context context, vx_reference ref, vx_status status, const vx_char string[])
+{
+    size_t len = strnlen(string, MAX_STRING_LENGTH);
+    if (len > 0) {
+        printf("%s", string);
+        if (string[len - 1] != '\n')
+            printf("\n");
+        fflush(stdout);
+    }
+}
+
+void read_input_digit(const int n, std::vector<float>& input_digit)
+{
+    std::ifstream file("../digits.txt");
```

Comment:
please check if this relative path is correct. Looks like digits.txt is in the same folder

Reply:
@rrawther the path is correct. The test executable is built in the build folder, and it's run from there. The digits is outside of the build folder.